src/struct_ls/F90_HYPRE_struct_bicgstab.c

```diff
@@ -138,6 +138,8 @@ hypre_F90_IFACE(hypre_structbicgstabsetprecond, HYPRE_STRUCTBICGSTABSETPRECOND)
     * The precond_id flags mean :
     * 0 - setup a smg preconditioner
     * 1 - setup a pfmg preconditioner
+    * 2 - setup a sparseMSG preconditioner
```

Comment:
Hi @victorapm .  We are not advertising the SparseMSG solver (we probably should delete it actually, but let's not do that quite yet), so I would not add this to the Fortran files.  If they already appear somewhere, I would go ahead and remove them instead.

Reply:
Ok, Rob! I'm fixing this with commit 24fb670